ord_schema/validations.py

```diff
@@ -496,6 +496,35 @@ def validate_reaction_input(message: reaction_pb2.ReactionInput):
                     ValidationWarning,
                 )
 
+    texture_type_to_state_of_matter = {
```

Comment:
document these values; consider creating a new enum with integer values

Reply:
added an `IntEnum` in https://github.com/open-reaction-database/ord-schema/pull/698/commits/c03e02687ae6831084669afc9a01eee6e3984d55